cobigen-templates/templates-devon4j/src/test/java/com/devonfw/cobigen/templates/devon4j/test/templates/testclasses/ReferenceEntity.java

```diff
@@ -0,0 +1,19 @@
+package com.devonfw.cobigen.templates.devon4j.test.templates.testclasses;
+
+import javax.persistence.*;
+
+@Entity
```

Comment:
Maybe include a javadoc

Reply:
I don't think putting javadoc in the test entities is necessary